nuclio_sdk/event.py

```diff
@@ -224,19 +224,5 @@ def deserialize(cls, data, kind=EventDeserializerKinds.msgpack_raw):
         """
         return kind.value.deserialize(data)
 
-    def compile_explicit_ack_message(self):
```

Comment:
I would make this use `QualifiedOffset.from_event(...). compile_explicit_ack_message` and raise a deprecation warning just to avoid any chance that we ruin working user code but this might be overkill so approved either way.